napari/layers/vectors/vectors.py

```diff
@@ -621,14 +621,27 @@ def edge_contrast_limits(
 
     @property
     def _view_face_color(self) -> np.ndarray:
-        """(Mx4) np.ndarray : colors for the M in view vectors"""
+        """(Mx4) np.ndarray : colors for the M in view triangles"""
```

Comment:
Why this change? This is always the M vectors in view, even with the arrow vis, right? Otherwise it would be the "M*3 triangles representing the M vectors" in that case?

Reply:
Ok, I see... It's just a bit confusing that we have some stuff working on vector basis and some on triangle basis, my bad :)